### PR TITLE
Tweaked how navigating to the settings page works

### PR DESCRIPTION
**Pull request by yaichenbaum**

### CODE REVIEW

**Comment by tsvietOK** [comment]
Here is how settings page looks at russian language.
image
Last string trimmed, full string is "Боковая панель с эффектом акрила"

**Comment by tsvietOK** [comment]
I want to suggest to put all controls on this page(Appearance) in one grid with first column width =Auto or put combo boxes and toggle buttons under their captions
Second, let's set same width for all combo boxes.

File: Files/Views/Pages/Settings.xaml [hunk 1]
```diff
@@ -40,6 +50,14 @@
                 </ResourceDictionary>
             </muxc:NavigationView.Resources>
 
+            <muxc:NavigationView.PaneHeader>
+                <TextBlock
+                    Margin="10,6"
```
Comment (tsvietOK) [inline Files/Views/Pages/Settings.xaml:55]: Here we need x:Uid for string translation

File: Files/Views/SettingsPages/Appearance.xaml [hunk 2]
```diff
@@ -45,7 +45,7 @@
                         Grid.Column="0"
                         VerticalAlignment="Center"
                         FontSize="16"
-                        Text="Theme" />
+                        Text="Choose an app mode" />
```
Comment (yaichenbaum) [inline Files/Views/SettingsPages/Appearance.xaml:48]: I would think the "Theme" is more descriptive, app mode can mean a number of other things such as tablet mode vs desktop mode...

File: Files/Views/SettingsPages/About.xaml [hunk 3]
```diff
@@ -67,8 +70,8 @@
                                 <TextBlock
                                     x:Uid="SettingsAboutSubmitFeedbackDescription"
                                     Foreground="Gray"
-                                    Text="Submit an issue report to provide the developers with information"
-                                    TextWrapping="Wrap" />
+                                    Text="Send the developers an issue report with more information"
```
Comment (yaichenbaum) [inline Files/Views/SettingsPages/About.xaml:73]: Unless this is changed in the en-US resource file, the text will show the changes made here.

File: Files/MultilingualResources/Files.de-DE.xlf [hunk 4]
```diff
@@ -575,7 +575,11 @@
           <source>MD5Hash:</source>
           <target state="translated">MD5Hash:</target>
         </trans-unit>
+        <trans-unit id="SettingsPageHeader.Text" translate="yes" xml:space="preserve">
```
Comment (tsvietOK) [inline Files/MultilingualResources/Files.de-DE.xlf:578]: There is already exists translated string Settings under id "SettingsNav.PaneTitle", so you can change name to "SettingsPageHeader.Text" and no need to translate it again

File: Files/Views/SettingsPages/Appearance.xaml (hunk 2 above)
Comment (yaichenbaum) [reply Files/Views/SettingsPages/Appearance.xaml:48]: I am changing it to "App theme", feel free to revert this change if you prefer it the other way.

Comment (duke7553) [reply Files/Views/SettingsPages/Appearance.xaml]: Hi, this change was to follow Microsoft guidelines for App Settings

Comment (tsvietOK) [reply Files/Views/SettingsPages/Appearance.xaml]: @duke7553 Another Microsoft app
image

Comment (yaichenbaum) [reply Files/Views/SettingsPages/Appearance.xaml]: While following the Microsoft Guidelines are good, I don't see any reason to follow this one as it seems to make the description less clear.

**Pull request merged**